<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorwindow</class>
 <widget class="QWidget" name="doctorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doctor</string>
  </property>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;BLACKHOLE HOSPITAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>711</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnProfile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Personal Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInfo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Check appointment information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlatform">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Communication Platform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCase">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Patient Case</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrescription">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit prescription and examination order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAttendance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Attendance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
